junit-jupiter-engine/src/main/java/org/junit/jupiter/engine/execution/InvocationInterceptorChain.java

```diff
@@ -104,10 +108,16 @@ public T proceed() throws Throwable {
 			return call.apply(interceptor, invocation);
 		}
 
+		@Override
+		public void skip() {
+			LOG.debug(() -> "The invocation is skipped");
```

Comment:
This should call `invocation.skip()` instead.

Reply:
done